pkg/apiserver/experiment/experiment.go

```diff
@@ -857,74 +868,73 @@ func (s *Service) deleteExperiment(c *gin.Context) {
 		return
 	}
 
-	uid := c.Param("uid")
-	if exp, err = s.archive.FindByUID(context.Background(), uid); err != nil {
-		if gorm.IsRecordNotFoundError(err) {
-			c.Status(http.StatusInternalServerError)
-			_ = c.Error(utils.ErrInvalidRequest.New("the experiment is not found"))
-		} else {
-			c.Status(http.StatusInternalServerError)
-			_ = c.Error(utils.ErrInternalServer.NewWithNoMessage())
+	uids := c.Param("uid")
+	uidSlice := strings.Split(uids, ",")
+	force := c.DefaultQuery("force", "false")
+
+	for _, uid := range uidSlice {
+		if exp, err = s.archive.FindByUID(context.Background(), uid); err != nil {
+			if gorm.IsRecordNotFoundError(err) {
+				joinErrString(&errStr, fmt.Sprintf("delete experiment uid (%s) error, because the experiment is not found", uid))
```

Comment:
[gin.Context.Error](https://pkg.go.dev/github.com/gin-gonic/gin@v1.6.3#Context.Error) seems to support multiple errors. What about recording errors separately instead of joining error strings?

Reply:
I'll check how to deal with it more reasonably.